src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -1188,6 +1199,19 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  private adjustCoordinates(): void {
+    const timelineActionsBandWidth = 404;
+    const scaleMaxWidth = 100;
+    const mapCanvas = document.getElementsByClassName('mapboxgl-canvas');
+    if (mapCanvas && mapCanvas.length > 0) {
+      const bbox = mapCanvas[0].getBoundingClientRect();
+      if (bbox) {
+        const width = bbox.width;
+        this.coordinatesHaveSpace = (width - timelineActionsBandWidth - scaleMaxWidth - 20) > 230;
```

Comment:
Why 20 ? Is it the dedired spacing between the coordinates and the timeline actions ?

Reply:
I fixed it. I added the toggle button width and 3 sm-margins